.github/workflows/release.yml

```diff
@@ -210,25 +210,24 @@ jobs:
           if [ "$TICKET_TYPE" == "Epic" ]; then
```

Comment:
1. In the curl command, the fields `self` and `assignee_url` have been removed from the API call. This means that the response will no longer include these fields.

2. The `extracted_data` variable has been modified to only extract the issue key and assignee name from the Jira API response. The `assignee_url` and `self` fields have been removed.

3. In the `awk` command, the variables for `issue_key`, `assignee`, and `assignee_url` have been updated. The `assignee` variable now appends `@` before the assignee's name.

4. The printf statement in the `awk` command has been modified to only include the issue key, status, and assignee. The `assignee_url` has been removed from the output.

Overall, these changes simplify the data extraction process from the Jira API response and modify the output format to exclude the `self` and `assignee_url` fields.